docs/py/opts.rst

```diff
@@ -27,6 +27,14 @@ The following compilation options are known.
    Extra arguments passed to ModelSim ``vlog`` command when compiling Verilog files.
    Must be a list of strings.
 
+``nvc.a_flags``
```

Comment:
Might we worth adding a "Global Options" or "Common Options" section, either at the beginning or the end of the page? Rather than repeating the `nvc.global_args` in both sections.

Reply:
My bad. It's actually two different options with the same name. Users might want to set "global sim options" but not "global elaboration options", or the other way round.
Therefore, it's good as is.